### PR TITLE
translate 2files into ko. (sourceRoot.md, strict.md)

### PR DESCRIPTION
**Pull request by seunghee114**
en :

- [sourceRoot.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/copy/en/options/sourceRoot.md)
- [strict.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/copy/en/options/strict.md)

번역 리뷰 요청합니다 : @yeonjuan @guyeol @dvlprsh

ref: [#910](https://github.com/microsoft/TypeScript-Website/issues/910)

감사합니다.

### CODE REVIEW

File: packages/tsconfig-reference/copy/ko/options/sourceRoot.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+display: "Source Root"
+oneline: "Sets the root path for debuggers to find the reference source code"
+---
+
+상대 소스 위치 대신 디버거가 TypeScript 파일을 찾을 위치를 지정합니다.
```
Comment (bumkeyy) [inline packages/tsconfig-reference/copy/ko/options/sourceRoot.md]:
```suggestion
소스의 상대 위치 대신 디버거가 TypeScript 파일을 찾을 위치를 지정합니다.
```
[제안]

File: packages/tsconfig-reference/copy/ko/options/sourceRoot.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+이 문자열은 경로 또는 URL을 사용할 수 있는 소스-맵 내에서 동사 그대로 처리됩니다:
```
Comment (bumkeyy) [inline packages/tsconfig-reference/copy/ko/options/sourceRoot.md]:
```suggestion
이 문자열은 경로 또는 URL을 사용할 수 있는 소스-맵 내에서 그대로 처리됩니다:
```
동사라는 뜻은 여기서 불필요한 것 같아요 :)

File: packages/tsconfig-reference/copy/ko/options/sourceRoot.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```json tsconfig
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "sourceRoot": "https://my-website.com/debug/source/"
+  }
+}
+```
+
+`index.js`는 `https://my-website.com/debug/source/index.ts`소스 파일을 갖게 될 것입니다.
```
Comment (bumkeyy) [inline packages/tsconfig-reference/copy/ko/options/sourceRoot.md]:
```suggestion
`index.js` 에 `https://my-website.com/debug/source/index.ts` 위치에 소스 파일이 있다고 명시해야 합니다.
```
이 뜻이 아닌가요?

File: packages/tsconfig-reference/copy/ko/options/strict.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+display: "Strict"
+oneline: "Enable all strict type checking options."
+---
+
+`strict` 플래그는 다양한 타입 검사를 가능하게 하여 프로그램 정확성을 더욱 보장합니다.
+strict 플래그를 켜는 것은 아래에 설명된 모든 _strict mode family_ 옵션을 활성화하는 것과 같습니다.
+필요에 따라 개별 strict mode family 검사를 끌 수 있습니다.
+
+향후 버전의 TypeScript는 이 플래그에서 추가적인 엄격한 검사를 도입할 수 있어, TypeScript를 업그레이드하면 프로그램에 새로운 타입 오류가 발생할 수도 있습니다.
+적절하고 가능하다면, 해당 플래그를 추가하여 동작을 비활성화합니다.
```
Comment (bumkeyy) [inline packages/tsconfig-reference/copy/ko/options/strict.md]:
```suggestion
적당히 가능할 때, 해당 플래그를 추가하여 동작을 비활성화합니다.
```
[제안]

**Comment by seunghee114** [comment]
@Bumkeyy 
제안주신대로 수정했습니다!
제가 잘못 이해한 부분도요!

감사합니다!!

**Comment by bumkeyy** [comment]
LGTM

**Comment by orta** [comment]
https://github.com/OSS-Docs-Tools/code-owner-self-merge/commit/6dab34d48c14e2f29ec4b3b4ef56660f4528522f

**Pull request merged**